README.md

```diff
@@ -23,17 +23,20 @@ It is currently expected that only the QoS/SLA tool will interact with these sum
 ## Running the docker image on Centos 7 and Ubuntu 16.04
 We recommend using the docker image to run the Accounting server and REST interface. As such, having Docker and docker-compose installed is a prerequisite.
 
-See [Ubuntu 16.04 Instructions](https://docs.docker.com/engine/installation/linux/ubuntulinux/) or [Centos 7 Instructions](https://docs.docker.com/engine/installation/linux/centos/) for details of how to install Docker.
+See [Ubuntu 16.04 Instructions](https://docs.docker.com/engine/installation/linux/docker-ce/ubuntu/) or [Centos 7 Instructions](https://docs.docker.com/engine/installation/linux/docker-ce/centos/) for details of how to install Docker. There is no requirement to undertake the 'Post-installation steps for Linux'.
 
 See [Install Docker Compose](https://docs.docker.com/compose/install/) for details of how to install docker-compose
```

Comment:
I'd refer to the link something like "See [the official Docker documentation]..." just to make it clear that this will link you to an external page.

Reply:
changed

Reply:
Ah, I was referring to line 28, not the line you actually changed in this PR. Line 26 was fine as it was.

Reply:
okay, changed back and then changed line 28.

I'll squash away the two commits when I eventually rebase